Debian/bin/lib/Cyrus/Docker/Command/makedocs.pm

```diff
@@ -0,0 +1,18 @@
+use v5.36.0;
+
+package Cyrus::Docker::Command::makedocs;
+use Cyrus::Docker -command;
+
+use Process::Status;
+
+sub abstract { 'make the docs site using Sphinx' }
+
+sub execute ($self, $opt, $args) {
+  my $root = "/srv/cyrus-imapd/docsrc";
+  chdir $root or die "can't chdir to $root: $!";
```

Comment:
I assume this either runs in a process that exits after `execute` returns, or in a context where everything is expected to chdir to where it needs to be, such that there's no need to restore the previous PWD before returning?

Reply:
Yeah.  Maybe I'll add more hygiene to this later, but generally `execute` runs and exits.